docs/tutorials/authenticate-api-requests.md

```diff
@@ -1,12 +1,45 @@
+---
+id: authenticate-api-requests
```

Comment:
Docs are still a bit messy, but I've restored the original api auth tutorial in contrib/ and added a new tutorial doc specifically about backend-to-backend authentication. Let me know your thoughts! In the meantime I'll focus on the other ongoing threads to try to get the API finalized, since this will have knock-on effects on the docs anyway.